crates/primitives-traits/src/account.rs

```diff
@@ -8,6 +8,7 @@ use reth_codecs::{reth_codec, Compact};
 use revm_primitives::{AccountInfo, Bytecode as RevmBytecode, JumpTable};
 use serde::{Deserialize, Serialize};
 
+pub use alloy_consensus::Account as AccountTrie;
```

Comment:
Sincerely, I don't think this name is idiomatic. I think we can found a better name for it 👍🏻